libs/utils/utils.js

```diff
@@ -485,10 +485,12 @@ export function decorateImageLinks(el) {
     const [source, alt, icon] = img.alt.split('|');
     try {
       const url = new URL(source.trim());
+      const hash = url.hash || '';
```

Comment:
Is this really needed? Per [MDN](https://developer.mozilla.org/en-US/docs/Web/API/URL/hash):
> If the URL does not have a fragment identifier, this property contains an empty string — "". Are there use cases where this could be `undefined` or `null`?

Reply:
Not that I found. I added it to be extra careful if there was something I missed. It probably can be removed.